x-pack/plugins/apm/readme.md

```diff
@@ -80,19 +80,19 @@ For debugging access Elasticsearch on http://localhost:9220` (elastic/changeme)
 
 ### API integration tests
```

Comment:
We should add something explaining why these different types of test exist and when we should write/run them.